### ISSUE TITLE
Support Python 3.7?

### ISSUE THREAD

**Issue by ebuildy** [xolox/python-coloredlogs#65]
Wondering why this is not supporting Python 3.7 ?

**Comment by mcepl** [comment]
What errors you have when it fails?

**Comment by mr-c** [comment]
`./scripts/generate-screenshots.py:130:33: W606 'async' and 'await' are reserved keywords starting with Python 3.7`

**Comment by hugovk** [comment]
3.7 support was added in https://github.com/xolox/python-coloredlogs/commit/9be101374b6b48b724bdb1b530cbf5d53a726963 (3.8 could also be added). Can you reproduce the error in the newest release?

**Comment by mr-c** [comment]
From a fresh checkout
```
$ make screenshots 
Already using interpreter /usr/bin/python3
Traceback (most recent call last):
  File "scripts/generate-screenshots.py", line 45, in <module>
    from capturer import CaptureOutput
ModuleNotFoundError: No module named 'capturer'
make: *** [Makefile:62: screenshots] Error 1
```

**Comment by xolox** [comment]
As of [coloredlogs 11.0](https://coloredlogs.readthedocs.io/en/latest/changelog.html#release-11-0-2020-02-14) (released yesterday) Python 3.7 is official supported:

- Older coloredlogs releases implicitly supported Python 3.7 already.
- The main thing I fixed yesterday was to add a workaround to the test suite for what I believe to be a regression in Python 3.7, for details on that refer to #66. Given that this concerns a regression in interpreter behavior for an obscure edge case in the test suite, it was never a real incompatibility.
- The `flake8` warnings [mentioned above](https://github.com/xolox/python-coloredlogs/issues/65#issuecomment-488310107) were fixed when #60 was merged.
- In any case `make screenshots` (`scripts/generate-screenshots.py`) generates screenshots which are also included in the version control repository. There's no reason for anyone but the project maintainer (me) to actually use this script. This explains why I didn't bother to properly document the additional dependencies.

In any case yesterday I also updated the `Makefile` to use Python 3 for local development and since then I noticed that `scripts/generate-screenshots.py` doesn't actually work on Python 3, so I will fix those incompatibilities and also document the additional Python installation requirements.
